source/aggregation-pipeline-builder.txt

```diff
@@ -207,6 +207,18 @@ your chosen view mode:
             stage card with a form that corresponds with the selected 
```

Comment:
[suggestion] Just a note that "New in version 1.14.0" appears to be missing a period.

Reply:
Just to double check, are you referring to it being missing after the last `0`?

Reply:
Yes!